www/js/input-page-anon.js

```diff
@@ -0,0 +1,698 @@
+	if(!date){
+		navigator.notification.alert("Please enter a date.");
+		return;
+	}
+
+
+	dateTime = date + " " + time;//Default seconds value to 00
+
+	siteID = $("#site option:selected").val();
+
+	if(siteID.localeCompare("default") === 0){
+		navigator.notification.alert("Please select a site");
+		return;
+	}
+	//if(oneline == true){
+	 //var showPasswordCheckboxIsChecked = document.getElementById("show-password").checked;
+	 //if(showPasswordCheckboxIsChecked){
+	//	sitePassword = $("#visible-password").val();
+	 //}else{
+	//	sitePassword = $("#hidden-password").val();
+	 //}
+	 //if(!sitePassword){
+	//	navigator.notification.alert("Please enter the site password");
+	//	return;
+	 //}
+	//}
+
+	surveyType = $(".survey-type option:selected").val();
+	if(surveyType.localeCompare("default")===0){
+		navigator.notification.alert("Please select a survey type.");
+		return;
+	}
+
+	circle = $("#circle option:selected").val();
+	if(circle.localeCompare("default")===0){
+		navigator.notification.alert("Please select a circle.");
+		return;
+	}
+
+	survey = $("#survey option:selected").val();
+	if(survey.localeCompare("default")===0){
+		navigator.notification.alert("Please select a survey.");
+		return;
+	}
+
+	//Trims leading and trailing white space from plant species field input
+	plantSpecies = $.trim($(".plant-species").val());
+	if(plantSpecies.length <= 0){
+		navigator.notification.alert("Please enter a plant species.");
+		return;
+	}
+	//Truncate plantSpecies that are more than 100 characters
+	else if(plantSpecies.length > 100){
+		plantSpecies = plantSpecies.substring(0, 100);
+	}
+
+	leafCount = $(".leaf-count").val();
+	if ( leafCount === "" || leafCount < 0 || leafCount > 200){
+		navigator.notification.alert("Please enter a leaf count between 0 and 200");
+		return;
+	}
+
+	switch ($("#herbivory-select").val()){
+		case "0":
+			herbivoryValue = 0;
+			break;
+		case "1":
+			herbivoryValue = 1;
+			break;
+		case "2":
+			herbivoryValue = 2;
+			break;
+		case "3":
+			herbivoryValue = 3;
+			break;
+		case "4":
+			herbivoryValue = 4;
+			break;
+		default :
+			navigator.notification.alert("Please select an herbivory score.");
+			return;
+	}
+
+	//if(!leafPhotoTaken){
+	//	navigator.notification.alert("Please take a leaf photo.");
+	//	return;
+	//}
+	//else{
+	//	leafImageURI = $("#leaf-photo").prop("src");
+	//}
+	//Check validity of site password
+	//Attempt to submit survey if password is valid
+	//navigator.notification.alert("SiteID: " + siteID +
+	//	"\nSite password: " +sitePassword);
+	//var online = navigator.onLine;
+	
+
+	db.transaction(function(tx){
+        tx.executeSql("INSERT INTO SURVEY VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)", ['survey',siteID,getURLParameter("userID"),getURLParameter("password"),circle,survey,dateTime,temperatures[temperature].min,temperatures[temperature].max,$(".notes").val(),plantSpecies,herbivoryValue,surveyType,parseInt(leafCount),"Mobile"]);
+            },function(error){
```

Comment:
this won't work properly for user id and pw, because there are no parameters in the url. Maybe you can hard code a id of 0 and and an empty pw, then fill it in when in online mode.